tests_async/system/requests/test_download.py

```diff
@@ -70,13 +73,44 @@ class CorruptingAuthorizedSession(tr_requests.AuthorizedSession):
     """
 
     EMPTY_HASH = base64.b64encode(hashlib.md5(b"").digest()).decode(u"utf-8")
+    EMPTY_MD5 = base64.b64encode(hashlib.md5(b"").digest()).decode(u"utf-8")
+    crc32c = _helpers._get_crc32c_object()
+    crc32c.update(b"")
+    EMPTY_CRC32C = base64.b64encode(crc32c.digest()).decode(u"utf-8")
 
     async def request(self, method, url, data=None, headers=None, **kwargs):
         """Implementation of Requests' request."""
         response = await tr_requests.AuthorizedSession.request(
             self, method, url, data=data, headers=headers, **kwargs
         )
-        response.headers[download_mod._HASH_HEADER] = u"md5={}".format(self.EMPTY_HASH)
+
+        temp = multidict.CIMultiDict(response.headers)
+        temp[download_mod._HASH_HEADER] = u"md5={}".format(self.EMPTY_HASH)
+        response._headers = temp
+        
+        """
```

Comment:
can we remove?

Reply:
Removed, as resolved with our changes to auth/resumable today.